packages/react-native-sdk/src/components/CallParticipantsInfoView.tsx

```diff
@@ -180,11 +179,12 @@ export const CallParticipantsInfoView = ({
           <View style={styles.header}>
             <View style={styles.leftHeaderElement} />
             <Text style={styles.headerText}>
-              Participants ({participantCount})
+              Participants ({participants.length})
```

Comment:
Partic. count originating from `useParticipantCount` has a certain lag, due to BE.
We want to have the most up-to-date count of partic. therefore we use `participants.length`